app/helpers/receiver_helper.rb

```diff
@@ -0,0 +1,7 @@
+module ReceiverHelper
+  MARKDOWN_PARSER = Redcarpet::Markdown.new(Redcarpet::Render::HTML)
+
+  def letter_html(receiver)
+    MARKDOWN_PARSER.render(receiver.letter)
+  end
```

Comment:
How about this pattern?

```ruby
def letter_html(receiver)
  parser.render(...)
end

private

def parser
  @_parser ||= Redcarpet::Markdown.new(Redcarpet::Render::HTML)
end
```

Won't it achieve the same effect? Seems cleaner than saving an object as a constant.

Reply:
Didn't know you could that, guess we learn something new everyday 🎉